index.js

```diff
@@ -64,8 +65,14 @@ module.exports = function autoRecord() {
       // Filter out blacklisted routes from being recorded and logged
       whitelist: xhr => {
```

Comment:
📓 docs on `whitelist`
- https://docs.cypress.io/api/commands/server.html#The-following-options-control-the-behavior-of-the-server-affecting-all-requests